tests/test_isamples_metadata.py

```diff
@@ -74,3 +75,18 @@ def test_geome_child_dicts_equal(geome_source_path, isamples_path):
         child_transformer = transformer.child_transformers[0]
         transformed_to_isamples_record = child_transformer.transform()
         _assert_transformed_dictionary(isamples_path, transformed_to_isamples_record)
+
+
+OPENCONTEXT_test_values = [
+    (
+        "../examples/OpenContext/raw/ark-28722-k2b570022.json",
+        "../examples/OpenContext/test/ark-28722-k2b570022-test.json",
```

Comment:
It make be good to have a few more test cases to verify mappings on items that have other extra attributes like "Has anatomical identification".